src/site/xdoc/server/config-blobstore.xml

```diff
@@ -53,14 +53,34 @@
                     The generated startup warning log can be deactivated via the <code>cassandra.blob.store.disable.startup.warning</code> environment
                     variable being positioned to <code>false</code>.
                 </dd>
+                Deduplication requires a garbage collector mechanism to effectively drop blobs. A first implementation
+                based on bloom filters can be used and triggered using the WebAdmin REST API. See
+                xref:distributed/operate/webadmin.adoc#_running_blob_garbage_collection[Running blob garbage collection].
 
+                In order to avoid concurrency issues upon garbage collection, we slice th blobs in generation, the two more recent
+                generation are not garbage collected.
+
+                *deduplication.gc.generation.duration*: Allow controlling the duration of one generation. Longer implies better deduplication
+                but deleted blobs will live longer. Duration, defaults on 30 days, the default unit is days.
+
+                *deduplication.gc.generation.family*: Every time the duration is changes, this integer counter must be incremented to avoid
+                conflicts. Defaults to 1.
                 <dt><strong>deduplication/enable</strong></dt>
                 <dd>Mandatory. Supported value: true and false.</dd>
                 <dd>If you choose to enable deduplication, the mails with the same content will be stored only once.</dd>
                 <dd>Warning: Once this feature is enabled, there is no turning back as turning it off will lead to the deletion of all</dd>
                 <dd>the mails sharing the same content once one is deleted.</dd>
-                <dd>This feature also requires a garbage collector mechanism to effectively drop blobs, which is not implemented yet.</dd>
-                <dd>Consequently, all the requested deletions will not be performed, meaning that blobstore will only grow.</dd>
+                <dd>This feature also requires a garbage collector mechanism to effectively drop blobs. A first implementation
+                    based on bloom filters can be used and triggered using the WebAdmin REST API. See
+                    <a href="manage-webadmin.html#Running_blob_garbage_collection">Running blob garbage collection</a>.
+                    In order to avoid concurrency issues upon garbage collection, we slice th blobs in generation, the two more recent
```

Comment:
```suggestion
                    In order to avoid concurrency issues upon garbage collection, we slice the blobs in generation, the two more recent
```